paper/paper.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'CeRULEo: Comprehensive utilitiEs for Remaining Useful Life Estimation methOds'
+tags:
+  - Python
+  - predictive maintenance
+  - remaining useful life  
+authors:
+  - name: Luciano Lorenti
+    orcid: 0000-0002-4041-449X
+    affiliation: 1
+    corresponding: true
+  - name: Gian Antonio Susto
+    orcid: 0000-0001-5739-9639
+    affiliation: 1
+affiliations:
+  - name:
+      Department of Engineering, University of Padova.
+      Italy
+    index: 1
+date: 20 Dic 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+`CeRULEo` is a Python package designed to train and evaluate regression models for predicting remaining useful life (RUL) of equipment. RUL estimation is a process that uses prediction methods to forecast the future performance of machinery and obtain the time left before machinery loses its operation ability.  The remaining useful life  estimation has been considered as a central 
```

Comment:
add full name for CeRULEo also in the text

Reply:
I added it at the beginning

`CeRULEo`, which stands for Comprehensive utilitiEs for Remaining Useful Life Estimation methOds, is ....